### PR TITLE
Update Winapp3.ini

### PR DESCRIPTION
**Pull request by APMichael**
Fixed various syntax and sorting errors.
Removed duplicate entry [Windows 7/8 Transcoded Wallpaper Cache \*].

### CODE REVIEW

File: Winapp3/Beta/Winapp3.ini [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
 [7-Zip Languages *]
-LangSecRef=Language Files
-DetectFile=%ProgramFiles%/7-zip
+Section=Language Files
+DetectFile=%ProgramFiles%\7-zip
 Default=False
 Warning=This will delete all languages excluding English.
-FileKey1=%ProgramFiles%/7-Zip/Lang|*.*
-ExcludeKey1=FILE|%ProgramFiles%/7-Zip/Lang|en.ttt
+FileKey1=%ProgramFiles%\7-Zip\Lang|*.*
```
Comment (ROCKNROLLKID) [inline Winapp3/Beta/Winapp3.ini]: Nice find. Perhaps this is a good suggestion for Winapp2ool to find?

File: Winapp3/Beta/Winapp3.ini [hunk 2]
```diff
@@ -582,6 +574,13 @@ FileKey2=%LocalAppData%\Black_Tree_Gaming\NexusClient.exe*\0.1*.*.*|*.*|REMOVESE
 FileKey3=%LocalAppData%\Black_Tree_Gaming\NexusClient.exe*\0.2*.*.*|*.*|REMOVESELF
 FileKey4=%LocalAppData%\Black_Tree_Gaming\NexusClient.exe*\0.3*.*.*|*.*|REMOVESELF
 
+[NVIDIA Installer2 Backup *]
```
Comment (ROCKNROLLKID) [inline Winapp3/Beta/Winapp3.ini]: Weird that Winap2ool didn't flag this, at least it didn't for me.

File: Winapp3/Beta/Winapp3.ini (hunk 1 above)
Comment (APMichael) [reply Winapp3/Beta/Winapp3.ini]: Yes, Winapp2ool could check Detects and Keys for the "wrong" slash (/).

**Comment by APMichael** [comment]
Thanks!

**Pull request merged**